source/clear-linux/tutorials/telemetry-e2e.rst

```diff
@@ -245,7 +245,7 @@ Once the installation is complete you can use your web browser and view the new
 You should see a web page similar to the one shown in figure 1:
 
 .. TODO fix links for figures
-.. figure:: telemetry-backend/figures/telemetry-backend-1.png
+.. figure:: ../guides/telemetrics/telemetry-backend-1.png
    :scale: 50 %
```

Comment:
@bd-dean Minor detail. For all documents that have figures, we need a `figures` directory added in which the two `.png` files reside. Will you please add that figures dir, move the .png files inside it, and update the two links again?

Reply:
done